airflow/executors/kubernetes_executor.py

```diff
@@ -23,6 +23,7 @@
 from queue import Empty
 from typing import Union
 from uuid import uuid4
+from six import iteritems
```

Comment:
Unfortunately, we do not use the six package in this project. Can you fix it some other way?

Reply:
It's possible also to use python-future pacakge or to make a Python version check manually.